### PR TITLE
feat: Add cosmosWallet util [DEV-2430]

### PR DESCRIPTION
**Pull request by DaevMithran**
Added three util functions to assist integration with cheqd sdk

1. `createCosmosPayerWallet(seed:string)`: helpful so that they need not understand the @cosmjs libraries

As we see multiple implementations where the client is signing the payload instead of passing the private key, these two utils will assist in creating the payload required to sign. Skipped update as it's the same as create payload
2. `async createMsgCreateDidDocPayloadToSign(DIDDocument, versionId)`
3. `createMsgDeactivateDidDocPayloadToSign(DIDDocument, versionId)`

### CODE REVIEW

**Comment by Eengineer1** [comment]
> _ No description provided. _

Could you add relevant description?

As in:
- What's it intended for?
- What are the relevant changes / features?

**Comment by DaevMithran** [comment]
> > _ No description provided. _
> 
> Could you add relevant description?
> 
> As in:
> 
> * What's it intended for?
> * What are the relevant changes / features?

Added the description now

**Comment by ankurdotb** [comment]
Task linked: [DEV-2430 Add Cosmos Wallet util](https://app.clickup.com/t/6600954/DEV-2430)

File: src/utils.ts [hunk 1]
```diff
@@ -25,12 +25,16 @@ import {
...
 } from '@stablelib/ed25519'
-import { sha256 } from '@cosmjs/crypto'
+import { DirectSecp256k1HdWallet, DirectSecp256k1Wallet } from '@cosmjs/proto-signing'
+import { EnglishMnemonic as _, sha256 } from '@cosmjs/crypto'
 import { v4 } from 'uuid'
 import {
     VerificationMethod as ProtoVerificationMethod,
     Service as ProtoService,
-} from "@cheqd/ts-proto/cheqd/did/v2/index.js"
+    MsgCreateDidDocPayload,
```
Comment (anikitinDSR) [inline src/utils.ts:34]: Do we need to support MsgUpdateDidDocPayload and MsgCreateResourcePayload also?

Comment (DaevMithran) [reply src/utils.ts:34]: Create did and update did have the same payload in sdk, Was not sure if we should create separate functions even if they are the same one's

Comment (anikitinDSR) [reply src/utils.ts:34]: I think it would be nice to have such function even if it's the same inside but it's more clear for understanding

**Review by anikitinDSR — APPROVED** [review]

**Pull request merged**